core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -454,14 +481,21 @@ private long processWalCommit(
                                 seqTxn,
                                 commitToTimestamp
                         );
+                        final long latency = microClock.getTicks() - start;
+                        long physicalRowCount = writer.getPhysicallyWrittenRowsSinceLastCommit();
+                        metrics.addApplyRowsWritten(rowCount, physicalRowCount, latency);
```

Comment:
I believe this is not same as 

```
metrics.addApplyRowsWritten(added, physicallyWrittenRowsSinceLastCommit, timeDelta);
```

since `rowCount` and `physicalRowCount` are accumulative counts, not per transaction but for a batch of transactions.